src/fiatconnect/saga.test.ts

```diff
@@ -0,0 +1,92 @@
+import { expectSaga } from 'redux-saga-test-plan'
+import { select } from 'redux-saga/effects'
+import {
+  fiatConnectCashInEnabledSelector,
+  fiatConnectCashOutEnabledSelector,
+} from 'src/app/selectors'
+import { fetchQuotes } from 'src/fiatconnect'
+import { handleFetchFiatConnectQuotes } from 'src/fiatconnect/saga'
+import {
+  fetchFiatConnectQuotes,
+  fetchFiatConnectQuotesCompleted,
+  fetchFiatConnectQuotesFailed,
+} from 'src/fiatconnect/slice'
+import { CICOFlow } from 'src/fiatExchanges/utils'
+import { getLocalCurrencyCode } from 'src/localCurrency/selectors'
+import { userLocationDataSelector } from 'src/networkInfo/selectors'
+import { CiCoCurrency } from 'src/utils/currencies'
+import { currentAccountSelector } from 'src/web3/selectors'
+import { mockFiatConnectQuotes } from 'test/values'
+
+jest.mock('src/fiatconnect')
```

Comment:
this mock works correctly unlike the `getPassword` one? Maybe because `fetchQuotes` has an explicit mock implementation in the test? 😕